package.json

```diff
@@ -11,6 +11,7 @@
   "license": "ISC",
   "dependencies": {
     "axios": "^1.3.4",
+    "axios-retry": "^3.4.0",
```

Comment:
This package is unused. You can remove it typing `npm uninstall axios-retry`

Reply:
this thing is gone now